js/customize-snapshots-admin.js

```diff
@@ -0,0 +1,73 @@
+( function( $ ) {
+	'use strict';
+
+	$( function() {
+		var $link, linkText, linkActions, dataSlug, initializeLink;
+
+		$link = $( '.snapshot-toggle-setting-removal' );
+		linkText = [ 'Remove setting', 'Restore setting' ];
```

Comment:
That's a good point. [This commit](https://github.com/xwp/wp-customize-snapshots/pull/84/commits/35d5bca1f03a9fb5cd1aef66a355b91dd30eba33) outputs the translated strings via PHP.